wp-modules/editor/js/src/blocks/pattern-block/PatternEdit.tsx

```diff
@@ -41,7 +41,11 @@ function PatternInspector( { pattern }: PatternInspectorProps ) {
 		<InspectorControls>
 			<Panel>
 				<PanelBody
-					title={ __( 'Pattern', 'pattern-manager' ) }
+					title={
+						pattern
+							? pattern.title
+							: __( 'Pattern', 'pattern-manager' )
```

Comment:
This isn't better, but here's another way to do this:

```js
title={ pattern?.title ?? __( 'Pattern', 'pattern-manager' ) }
```

Don't change this.

I just thought I'd share it, as I'm learning about the `??` operator.

Reply:
Thanks @kienstra! 

Appreciate the info too!

Reply:
Thanks!